src/components/DynamicFormContainer/DynamicFormLayout.tsx

```diff
@@ -51,7 +51,8 @@ export const DynamicFormLayout: FunctionComponent = () => {
     const nextForms = [...forms];
     nextForms.splice(activeFormIndex, 1);
     setForms(nextForms);
-    const nextActiveFormIndex = nextForms.length > 0 ? nextForms.length - 1 : undefined;
+    const nextActiveFormIndex =
```

Comment:
I discussed with jerry ytd and he said would go to next instead of previous. Please check with him

Reply:
I have talked to jerry about it, the current one is going next form. so example if u delete the current form on page 3, it will show the document that is on page 4.